docs/docsite/rst/dev_guide/developing_modules_best_practices.rst

```diff
@@ -67,7 +67,7 @@ Python tips
 Importing and using shared code
 ===============================
 
-* Use shared code whenever possible - don't reinvent the wheel. Ansible offers the ``AnsibleModule`` common Python code, plus :ref:`utilities <appendix_module_utilities>` for many common use cases and patterns.
+* Use shared code whenever possible - don't reinvent the wheel. Ansible offers the ``AnsibleModule`` common Python code, plus :ref:`utilities <developing_module_utilities>` for many common use cases and patterns.
```

Comment:
add note about 'shared docs' i.e doc_fragments?

Reply:
addressed in 584f57483bbb5501037da59219fecafb820bbbc2